preview/MsixCore/msixmgr/MsixRequest.cpp

```diff
@@ -176,7 +176,12 @@ HRESULT MsixRequest::ProcessAddRequest()
         }
         if (FAILED(hr) && currentHandler.errorMode != IgnoreAndProcessNextHandler)
         {
-            m_msixResponse->SetErrorStatus(hr, L"Failed to process add request");
+            // Set response object with generic response if not explicitly caught and set by handlers already
+            if (!m_msixResponse->GetHResultTextCode())
+            {

```

Comment:
put this check inside the SetErrorStatus and use SUCCEEDED()

Reply:
Moved to inside SetErrorStatus and compared with SUCCEEDED().